build-tools/packages/build-cli/src/commands/release/report-unreleased.ts

```diff
@@ -186,6 +196,9 @@ async function updateReportVersions(
 
 function extractBuildNumber(version: string): number {
 	const versionParts: string[] = version.split("-");
+	if (version.includes("test")) {
```

Comment:
A comment here explaining why we need to do this and showing an example of how versions look in test/ branches would be nice.

Reply:
Ideally we would have a function in version-tools that understod test versions. Not sure if that's i scope for this change but if there are even two of these sorts of checks then we should spend the time to build the testable function.